include/openssl/http.h

```diff
@@ -38,63 +36,64 @@ typedef BIO *(*OSSL_HTTP_bio_cb_t)(BIO *bio, void *arg, int connect, int detail)
 #define HTTP_DEFAULT_MAX_LINE_LENGTH (4 * 1024)
 #define HTTP_DEFAULT_MAX_RESP_LEN (100 * 1024)
```

Comment:
Good point to change this. Yet why in a separate PR rather than in a (separate) fixup commit here?
This PR is about all the HTTP client API changes, and there would be some conflicts if done elsewhere.
Anyway these two constants are not used often.

Reply:
Ah, just saw your other comment regarding you wanna approve right  away, so deferring this :)

Reply:
Even better!